curriculum/locales/english/build-a-video-game-marketplace-blockchain.md

```diff
@@ -399,7 +399,7 @@ assert.equal(
 );
 ```
 
-Running `node sell-item.js <address-privateKey> <item>` should add a transaction at the end of the `transactions.json` array with the correct `buyerAddress`, `sellerAddress`, `price`, `itemSold`, and `signature` properties
+Running `node sell-item.js <address-privateKey> <item>` should add a transaction at the end of the `transactions.json` array with the correct `buyerAddress`, `sellerAddress`, `price`, `itemSold`, and `signature` properties. Note: You may need to pass the item check test right below this for this to pass
```

Comment:
Suggestion: It might be better to mention the test number, because, if a Camper is passing that test, it will not be the one below.